### PR TITLE
Added list of moves

### LINKED ISSUE #1: Figure out rest of moves
Complete list of possible moves

Also add issues to implement resolvers for each move

### PR DESCRIPTION
**Pull request by JustinHuPrime**
We now have a list of moves the client can send to the server

I removed some stretch goal stuff from the board and balance files

Closes #1

### CODE REVIEW

File: src/common/Board.hs [hunk 1]
```diff
@@ -9,12 +9,6 @@ data Entity
     AstroObj Int Vec2 String Double Double
   | --- Asteroid cluster with some id, some x, y position
     AsteroidCluster Int Vec2
-  | --- Mine with some id, some x, y position and some dx, dy velocity, and some lifetime
-    Mine Int Vec2 Vec2 Int
-  | --- Torpedo with some id, some x, y position and some dx, dy velocity, and some lifetime
-    Torpedo Int Vec2 Vec2 Int
-  | --- Nuke with some id, some x, y position and some dx, dy velocity, and some lifetime
-    Nuke Int Vec2 Vec2 Int
```
Comment (wyndro) [inline src/common/Board.hs:17]: So we are making ordinants stretch goals? I feel like that is a good choice.

File: src/common/Move.hs [hunk 2]
```diff
@@ -5,5 +5,6 @@ import Util
 data Move
   = --- thust the ship with given id the given amount
     Thrust Int Vec2
-
--- TODO: fill this in
\ No newline at end of file
+  | --- launch a mine from the given ship
+    Attack Int Int
```
Comment (wyndro) [inline src/common/Move.hs:9]: Please document what the two Ints are. I think one is the id of the ship but I don't know what the other is.

**Review by wyndro — CHANGES_REQUESTED** [review]
Just a small request for some documentation in the moves

File: src/common/Move.hs (hunk 2 above)
Comment (JustinHuPrime) [reply src/common/Move.hs:9]: Whoops! This is fixed in later commits, but the comment isn't. Fix committed.

**Review by wyndro — APPROVED** [review]

**Review by JohannCooper — APPROVED** [review]

**Pull request merged**